src/lib/specialistBios/index.ts

```diff
@@ -0,0 +1,55 @@
+import {
```

Comment:
Why are we storing a slice of the schema in the `lib` folder? The schema goes in `schema`, with the rest of our schema.

Reply:
✅